packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/lambda/integ.invoke.ts

```diff
@@ -49,6 +49,18 @@ const checkJobState = new LambdaInvoke(stack, 'Check the job state', {
   resultSelector: {
     status: sfn.JsonPath.stringAt('$.Payload.status'),
   },
+  payload: sfn.TaskInput.fromObject({
```

Comment:
Can you add an assertion here to assert that the state machine execution
succeeded? The reason that the assertion is needed is because this test
currently only tests that the state machine is able to be deployed, but there is
not any validation that the state machine is actually valid and will execute.

You can find an example here
https://github.com/corymhall/aws-cdk/blob/53d24f1a1357410746052bbf25b5b89b985f7b80/packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/aws-sdk/integ.call-aws-service-sfn.ts#L38-L49

You can probably also have the lambda function validate that all those fields
are set.